db/migrate/20210808180355_add_seats_to_cinema_hall.rb

```diff
@@ -1,7 +0,0 @@
-# frozen_string_literal: true
-
-class AddSeatsToCinemaHall < ActiveRecord::Migration[6.1]
-  def change
-    add_column :cinema_halls, :seats, :text, array: true, default: []
-  end
-end
```

Comment:
Unfortunately removing existing migrations is considered a bad practice. See your schema file you still have `cinema_halls` reference. 

For now, you can just remove it and re-create `db` but when on production it would cause an error because of these missing migrations. For that, we would create a separate migration for removing this column not just removing the file.